### PR TITLE
[Method References] LPS-163567 Convert method references to field references in module portal-search-elasticsearch7-impl | part 4

### PR DESCRIPTION
**Pull request by W2jx**
Hi @tinatian @dantewang , I convert method references to field references in module portal-search-elasticsearch7-impl.

### CODE REVIEW

**Comment by vicnate5** [comment]
Testing new team. @liferay/qa-core-infra let me know in slack if you got this message

**Comment by catarineaguiar** [comment]
@W2jx no regressions found.

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchSearchEngine.java [hunk 1]
```diff
@@ -144,6 +144,16 @@ public Collection<Long> getIndexedCompanyIds() {
 		return companyIds;
 	}
 
+	@Override
+	public IndexSearcher getIndexSearcher() {
```
Comment (dantewang) [inline modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchSearchEngine.java:148]: @W2jx After removing `setXXX` and implementing `getXXX`, the `setXXX` methods and `_xxx` fields in `BaseSearchEngine` are no longer used (even the other implementation doesn't use it). I think we should remove these `getXXX`, `setXXX` and `_xxx` from `BaseSearchEngine`.

**Review by dantewang — CHANGES_REQUESTED** [review]

File: modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchSearchEngine.java (hunk 1 above)
Comment (W2jx) [reply modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/ElasticsearchSearchEngine.java:148]: I will fix it.

**Pull request closed without merging**